src/components/Alert/index.tsx

```diff
@@ -1,47 +1,32 @@
-import { useEffect } from 'react'
-import { useAppState, useDispatch } from '../../contexts/providers'
+import { useState } from 'react'
 import { AlertPanel } from './styled'
 import i18n, { currentLanguage } from '../../utils/i18n'
 import { dayjs, parseSimpleDateNoSecond } from '../../utils/date'
 import SimpleButton from '../SimpleButton'
-import { ComponentActions } from '../../contexts/actions'
 import { AppCachedKeys } from '../../constants/cache'
 import { IS_MAINTAINING } from '../../constants/common'
 import styles from './styles.module.scss'
+import { useStatistics } from '../../services/ExplorerService'
+import { createGlobalState, createGlobalStateSetter, useGlobalState } from '../../utils/state'
 
 const FIFTEEN_MINUTES = 15 * 60 * 1000
 
+const globalMaintenanceMsg = createGlobalState<[string, string] | []>([])
+
+export const setMaintenanceMsg = createGlobalStateSetter(globalMaintenanceMsg)
+
 const Alert = () => {
-  const dispatch = useDispatch()
-  const {
-    app: { appErrors },
-    components: { maintenanceAlertVisible },
-    statistics: { reorgStartedAt },
-  } = useAppState()
-  const [startTime, endTime] = appErrors[2].message
+  const { reorgStartedAt } = useStatistics()
```

Comment:
The message `maintenance starts from xxx, ends at xxx` can be removed because it's not used anymore. Now `maintenance status` is toggled by a runtime environment set in vercel

Reply:
removed

Reply:
I see this code is still here

Reply:
> I see this code is still here

I think what you may be seeing is a cached result, the branch you are looking at from here has already been removed:
https://github.com/WhiteMinds/ckb-explorer-frontend/blob/refactor/433_RemoveGlobalState/src/components/Alert/index.tsx

Reply:
> > I see this code is still here
> 
> I think what you may be seeing is a cached result, the branch you are looking at from here has already been removed: [WhiteMinds/ckb-explorer-frontend@`refactor`/433_RemoveGlobalState/src/components/Alert/index.tsx](https://github.com/WhiteMinds/ckb-explorer-frontend/blob/refactor/433_RemoveGlobalState/src/components/Alert/index.tsx?rgh-link-date=2023-09-28T12%3A15%3A35Z)

I can see a commit (https://github.com/WhiteMinds/ckb-explorer-frontend/commit/4aecf4050e9af7bdd9e6c8e54b8d10b32addd5f0) was submitted but 

```js
const { reorgStartedAt } = useStatistics()
```

still left

Reply:
Oh, so this property is no longer in use either.
I just removed it in https://github.com/Magickbase/ckb-explorer-frontend/pull/103/commits/2721e9b38614b2e17d0589a61833be409c802869 .